project_template/{{cookiecutter.project_repo_name}}/config/config.yaml

```diff
@@ -8,41 +8,18 @@ global:
   env:
     PROJECT_SLUG: '{{cookiecutter.project_slug}}'
 {% raw %}
-    ENVIRONMENT_NAME: {{ env.ENVIRONMENT_NAME }}
     ENVIRONMENT: '{{ env.ENVIRONMENT | default("local", True) }}'
```

Comment:
@lbenc135  Do we need this variable? Is it local by default?

Reply:
we don't, yes it's local by default